components/array-object-table/src/stories.js

```diff
@@ -52,28 +53,34 @@ examples.add('skipEmtpyRows and hideWithNoValues (table is hidden if no rows)',
   ];
   const title = ['Heading'];
 
-  return <ArrayObjectTable fields={fields} array={array} title={title} skipEmptyRows hideWithNoValues/>;
+  return (<ArrayObjectTable
+    fields={fields}
+    array={array}
+    title={title}
+    skipEmptyRows
+    hideWithNoValues
+  />);
 });
 
 examples.add('With object transform and default transform', () => {
   const fields = [
     { key: 'one', heading: 'One' },
-    { key: 'two', heading: 'Two', transform: value => value ? value.toLowerCase() : '' },
+    { key: 'two', heading: 'Two', transform: value => (value ? value.toLowerCase() : '') },
```

Comment:
not sure i see the value in the brackets here

Reply:
This one is "Arrow function used ambiguously with a conditional expression" (`no-confusing-arrow`). Again will take a look at options...

Reply:
yeah - as i said this one i'm not sure about - there's potentially value in that rule i guess, especially if the code in the ternary gets more complex and spread over multiple lines

Reply:
Hm.. yes, there isn't any multi-line setting. We could override this rule if you like (turn it off)?